src/components/transactionsV2/walletTransactionsV2/walletTransactionsV2.js

```diff
@@ -198,8 +209,14 @@ class WalletTransactionsV2 extends React.Component {
         }
 
         <TabsContainer>
-          <WalletTab tabName={this.props.t('Wallet')}
+          <WalletTab tabName={t('Wallet')}
             {...overviewProps}/>
+          {/* account.isDelegate && delegate.txDelegateRegister
```

Comment:
Here too.... or I dont know if it is because you need it for later

Reply:
This one is because the delegate tab will be used on the wallet page, but not right now.